easybuild/easyconfigs/j/jax/jax-0.7.0-gfbf-2025a.eb

```
@@ -0,0 +1,97 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+# Updated by: Alex Domingo (Vrije Universiteit Brussel)
+# Updated by: Thomas Hoffmann (EMBL Heidelberg)
+# Updated by: Pavel Tománek (INUITS)
+
+easyblock = 'PythonBundle'
+
+name = 'jax'
+version = '0.7.0'
+
+homepage = 'https://jax.readthedocs.io/'
+description = """Composable transformations of Python+NumPy programs:
+differentiate, vectorize, JIT to GPU/TPU, and more"""
+
+toolchain = {'name': 'gfbf', 'version': '2025a'}
+
+builddependencies = [
+    ('Bazel', '7.4.1', '-Java-21'),
+]
+
+dependencies = [
+    ('Python', '3.13.1'),
+    ('SciPy-bundle', '2025.06'),
+    ('absl-py', '2.3.1'),
+    ('flatbuffers-python', '25.2.10'),
+    ('ml_dtypes', '0.5.1'),
```

Those EasyConfigs are still missing, so that's why boegelbot fails silently.
The CI complains as well.

`absl-py` and `flatbuffers-python` have a version for `2025b` and can probably easily be backported. `ml_dtypes` is not updated yet.

---

Oh, sorry, I forgot to add them here